apps/flows/src/app/transfer-and-pay-flow/transfer-and-pay-modal/transfer-and-pay-modal.component.ts

```diff
@@ -0,0 +1,179 @@
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { COMPONENT_PROPS, ModalConfig, ModalController } from '@kirbydesign/designsystem';
+import { ChooseReceiverComponent } from '../choose-receiver/choose-receiver.component';
+import { ChooseOwnAccountComponent } from '../choose-own-account/choose-own-account.component';
+import { OtherService } from '../other.service';
+import { OwnAccountService } from '../own-account.service';
+import { OwnAccount } from '../own-account';
+import { Account } from '../account';
+import { ChooseDateComponent } from '../choose-date/choose-date.component';
+import { ChooseTextAndMessageComponent } from '../choose-text-and-message/choose-text-and-message.component';
+import { TextAndDateService } from '../text-and-date.service';
+import { TransferRegisteredComponent } from '../transfer-registered/transfer-registered.component';
+import { DetailsComponent } from '../details/details.component';
+import { VerifyService } from '../verify.service';
+import { Details } from '../details';
+
+@Component({
+  selector: 'flows-transfer-and-pay-modal',
+  templateUrl: './transfer-and-pay-modal.component.html',
+  styleUrls: ['./transfer-and-pay-modal.component.scss'],
+})
+export class TransferAndPayModalComponent implements OnInit {
+  constructor(
+    private modalController: ModalController,
+    private otherService: OtherService,
+    private ownAccountService: OwnAccountService,
+    private textAndDateService: TextAndDateService,
+    private verifyService: VerifyService
+  ) {}
+
+  @ViewChild('input') input: ElementRef<HTMLInputElement>;
+  currency: string;
+  currencyEntered: boolean = false;
+  receiverChosen: boolean = false;
+  isDisabled: boolean;
+  transferRegistered: boolean = false;
+  selectedAccount: OwnAccount[];
+  selectedReceiver: Account[];
+  date: string;
+  text: string;
+  message: string;
+  verifiedDetails: Details;
+
+  ngOnInit(): void {
+    this.otherService.getReceiverBoolean$().subscribe((boolean) => (this.receiverChosen = boolean));
```

Comment:
`boolean` doesn´t tell my what you are assigning

Reply:
Renamed